homeassistant/components/unifi_access/coordinator.py

```diff
@@ -0,0 +1,101 @@
+"""Coordinators for the UniFi Access integration."""
+
+from asyncio import CancelledError, Task
+from contextlib import suppress
+import logging
+
+import aiohttp
+import uiaccessclient
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class UniFiAccessDoorCoordinator(DataUpdateCoordinator[dict[str, uiaccessclient.Door]]):
```

Comment:
Can we maybe just import the object instead of just the lib?
```suggestion
class UniFiAccessDoorCoordinator(DataUpdateCoordinator[dict[str, Door]]):
```

Reply:
Yep!

Reply:
This is fixed.